### ISSUE TITLE
lang attribute is associated with HTML, not CSS

### ISSUE THREAD

**Issue by r12a** [w3c/pronunciation#45]
3.2 Voice Family/Gender/CSS
https://www.w3.org/TR/pronunciation-gap-analysis/#voice-family-gender

> lang attribute can be applied to the document itself or to elements. (WCAG) (AT Supported: some)

Perhaps the title of this subsection should be HTML?
